src/main/java/io/stargate/sgv2/jsonapi/service/embedding/configuration/EmbeddingProvidersConfig.java

```diff
@@ -189,28 +189,49 @@ public String toString() {
     interface RequestProperties {
 
       /**
-       * The maximum number of retries to attempt before failing the request.
+       * Specifies the maximum number of attempts before failing. Default is 3 (1 request + 2
+       * retries).
        *
-       * @return The maximum number of retries to attempt before failing the request.
+       * @return The maximum number of attempts before failing.
        */
       @WithDefault("3")
-      int maxRetries();
+      int atMostRetries();
```

Comment:
nit: technically this would be more like "atMostAttempts()", not retries, if comment is correct (value is one more than maximum retries).
But can be left as-is as comment explains specific logic.